assets/js/modules/adsense/settings/settings-setup-incomplete.js

```diff
@@ -48,7 +47,7 @@ export default function SettingsSetupIncomplete() {
 				className="googlesitekit-settings-module__edit-button"
 				href={
 					// TODO: Replace this function with something datastore-driven.
-					getReAuthURL( 'adsense', true )
+					useSelect( ( select ) => select( STORE_NAME ).getAdminReauthURL() )
```

Comment:
This should be better assigned to a variable further above with the other hooks instead of inline here.

Reply:
Variable added - adminReauthURL